code/modules/antagonists/cult/team_cult.dm

```diff
@@ -0,0 +1,485 @@
+									All dead cultists should be taken to medbay or robotics for immediate revival and deconversion. \
+									Non-security staff may defend themselves, but should prioritize leaving any areas with cultists and reporting the cultists to security. \
+									Self defense permits non-security staff to use lethal force as a last resort. Hunting down cultists may make you liable for a manslaughter charge. \
+									Any access granted in response to the paranormal threat should be reset. \
+									Any and all security gear that was handed out should be returned. Finally, all weapons (including improvised) should be removed from the crew.",
+									"Central Command Higher Dimensional Affairs", 'sound/AI/commandreport.ogg')
+/**
+ * This is a magic fuckin proc that takes a proc_ref, and calls it on all the human cultists.
+ * Created so that we don't make 1000 timers, and I'm too lazy to make a proc for all of these.
+ * Used in callbacks for some *magic bullshit*.
+ */
+/datum/team/cult/proc/all_members_timer(proc_ref_to_call)
+	for(var/datum/mind/M in members)
+		if(!ishuman(M.current))
+			continue
+		var/datum/antagonist/cultist/cultist = M.has_antag_datum(/datum/antagonist/cultist)
+		if(cultist)
+			call(cultist, proc_ref_to_call)() // yes this is a type proc ref passed by a callback, i know its deranged
+
+/datum/team/cult/proc/is_convertable_to_cult(datum/mind/mind)
+	if(!mind)
+		return FALSE
+	if(!mind.current)
+		return FALSE
+	if(IS_SACRIFICE_TARGET(mind))
+		return FALSE
+	if(mind.has_antag_datum(/datum/antagonist/cultist))
+		return TRUE //If they're already in the cult, assume they are convertable
+	if(HAS_MIND_TRAIT(mind.current, TRAIT_HOLY))
+		return FALSE
+	if(ishuman(mind.current))
+		var/mob/living/carbon/human/H = mind.current
+		if(ismindshielded(H)) //mindshield protects against conversions unless removed
+			return FALSE
+	if(mind.offstation_role)
+		return FALSE
+	if(issilicon(mind.current))
+		return FALSE //can't convert machines, that's ratvar's thing
+	if(isguardian(mind.current))
+		var/mob/living/simple_animal/hostile/guardian/G = mind.current
+		if(IS_CULTIST(G.summoner))
+			return TRUE //can't convert it unless the owner is converted
+	if(isgolem(mind.current))
+		return FALSE
+	if(isanimal(mind.current))
+		return FALSE
+	return TRUE
+
+/**
+  * Decides at the start of the round how many conversions are needed to rise/ascend.
+  *
+  * The number is decided by (Percentage * (Players - Cultists)), so for example at 110 players it would be 11 conversions for rise. (0.1 * (110 - 4))
+  * These values change based on population because 20 cultists are MUCH more powerful if there's only 50 players, compared to 120.
+  *
+  * Below 100 players, [CULT_RISEN_LOW] and [CULT_ASCENDANT_LOW] are used.
+  * Above 100 players, [CULT_RISEN_HIGH] and [CULT_ASCENDANT_HIGH] are used.
+  */
+/datum/team/cult/proc/cult_threshold_check()
```

Comment:
You should probably add a notification to cultists at the 20 minute mark to indicate how many members are actually needed, or at least that the number has been updated

Reply:
Cult members can use their "study" action button to see how many more left to rise/ascend at any time, and this is only called a second time 2 minutes into the game, after people have latejoined. I think its fine

Reply:
Sounds fair